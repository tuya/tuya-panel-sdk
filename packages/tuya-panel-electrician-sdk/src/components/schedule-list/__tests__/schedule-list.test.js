import React from 'react';
import { View } from 'react-native';
import { shallow } from 'enzyme';
import ScheduleList from '../index';

describe('ScheduleList components', () => {
  it('basic render', () => {
    const wrapper = shallow(<ScheduleList dpCodes={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
