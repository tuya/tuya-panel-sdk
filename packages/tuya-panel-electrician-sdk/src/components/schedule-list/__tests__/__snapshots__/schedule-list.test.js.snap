// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleList components basic render 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
    }
  }
>
  <TYFlatList
    data={Array []}
    flatListRef={null}
    separatorStyle={null}
    useART={false}
  />
</View>
`;
